gits/test/terminal_test.py

```diff
@@ -582,24 +582,24 @@ def test_esc_da(self):
         pass
 
     def test_cap_ri(self):
-        """The terminal should scroll down by 1 position when terminal's `top`
-        was chosen as maximum between terminal's `top` and terminal's `cur_y`.
-        """
-        
-        # Cursor at left-most position, `top` and `cur_y` equal 0.
-        s = ['a'] * self._terminal._right_most
+        """The terminal should have the possibility to scroll text down. """
```

Comment:
@dshil IMO the information should be added to the `_cap_ri` docstring. What do you think?